apps/dashboard/src/data/action-types.js

```diff
@@ -19,6 +19,8 @@ export const POLL_UPDATE = 'POLL_UPDATE';
 export const PROJECT_SAVE = 'PROJECT_SAVE';
 export const PROJECT_UPDATE = 'PROJECT_UPDATE';
 export const PROJECT_SAVE_ERROR = 'PROJECT_SAVE_ERROR';
+export const PROJECT_LOADED = 'PROJECT_LOADED';
+export const PROJECT_CHANGE = 'PROJECT_CHANGE';
```

Comment:
This is a little less descriptive than I'd have hoped, can we change it to `PROJECT_STATUS_UPDATE` ?

We can then create an 'enum' for the different status values as required by the frontend, so in addition of updating the actual project object, these could be stored under a separate reducer key.

As for where the enum should go: `/src/util/project/status.js` - we already have some in `/src//util/user/...` if you need a reference.

Suggested values:

`DRAFT`
`DRAFT_WITH_UNSAVED_CHANGES`
`PUBLIC`
`PUBLIC_WITH_UNSAVED_CHANGES`
`PUBLIC_WITH_UNPUBLISHED_CHANGES`

A bit lengthy but I think it gives the right picture. If anything they'll be easy to rename once we start using them.